notebooks/import.py

```diff
@@ -217,11 +217,13 @@ def create_points(collection_id, histogram):
            '(collection_id, bucket, proportion, count, rank) '
            'VALUES %s')
 
-    for rank, kv in enumerate(histogram.iteritems(), 1):
-        k, v = kv[0], kv[1].asDict()
+    rank = 0
+    for k in sorted(histogram.keys()):
```

Comment:
Any particular reason to prefer manual iteration vs `enumerate`?

Reply:
Nope, just how it came out of my brain b/c of the change to sorting the keys. I'll update.